src/server/index.spec.tsx

```diff
@@ -106,5 +108,102 @@ describe(__filename, () => {
         expect(response.text).toContain(expectedHTML);
       });
     });
+
+    describe('NODE_ENV=development', () => {
+      let proxy: express.Application;
+      let proxyServer: http.Server;
+
+      const devEnv = {
+        NODE_ENV: 'development',
+        REACT_APP_CRA_PORT: 60123,
+      };
+
+      beforeEach(() => {
+        proxy = express();
+        proxyServer = proxy.listen(devEnv.REACT_APP_CRA_PORT);
```

Comment:
Maybe this is obvious because of the env var but I was confused at how `createServer` was able to connect to this. I would suggest adding a comment saying something like: This is meant to simulate the create-react-app dev server